src/envoy/utils/grpc_transport.h

```diff
@@ -25,6 +25,9 @@
 
 #include "envoy/upstream/cluster_manager.h"
 #include "include/istio/mixerclient/client.h"
+#include "src/envoy/utils/header_update.h"
+
+using ::istio::mixer::v1::Attributes;
```

Comment:
per c++ style guide,  header file should not use "namespace using".  have to put in the long namespace.

Reply:
done